frontend-react/src/webreceiver-utils.ts

```diff
@@ -6,10 +6,13 @@ const groupToOrg = (group: String | undefined): string => {
     // in order to replace all instances of the underscore we needed to use a
     // global regex instead of a string. a string pattern only replaces the first
     // instance
+    const isSender = group?.startsWith(PERMISSIONS.SENDER);
```

Comment:
Maybe consistent use of PERMISSIONS.SENDER vs `${PERMISSIONS.SENDER}_` check? I like including the `_` in the group check. I could see using a single `const` at the top of the function and use it for the `startsWith` and the `.replace`